docs/site/Deploy-for-GDPR-readiness.md

```diff
@@ -0,0 +1,172 @@
+* [ibm.com/GDPR website](http://ibm.com/GDPR)
+
+
+___________________________________________________________________________________________________
+### Product Configuration - considerations for GDPR Readiness
+#### Offering Configuration
+
+The following sections provide considerations for configuring LoopBack to help your organization with GDPR readiness.
+
+##### Configuration to support data handling requirements
+The GDPR legislation requires that personal data is strictly controlled and that the
+integrity of the data is maintained. This requires the data to be secured against loss
+through system failure and also through unauthorized access or via theft of computer equipment or storage media.
+
+**LoopBack Overview**
+
+_Please note that LoopBack 4 is in developer preview and not ready for production.  For production, use [LoopBack 3](https://loopback.io/doc/en/lb3/index.html)._
+
+LoopBack is an open source Node.js framework that allows you to create REST APIs and connect to the backend resources, such as database and Web services.
+- Users can download LoopBack anonymously and use LoopBack without creating any accounts.
+- LoopBack itself is not runnable. The close analogy is Spring Framework or IBM Node.js SDK.
+- Customers create applications using LoopBack tooling and runtime.
+- Since LoopBack allows users to connect to backend resources that might require credentials to access, users can create a LoopBack application in such way that the credentials are stored in the configuration files as part of their LoopBack application.
+
+It is important to note that it is developer's responsibility to design and configure their LoopBack application using the [GDPR Privacy By Design](https://www.eugdpr.org/more-resources-1.html) as guidelines.
+
+##### Configuration to support Data Privacy
+It is developer's responsibility to design and configure their LoopBack application using the [GDPR Privacy By Design](https://www.eugdpr.org/more-resources-1.html) as guidelines to support Data Privacy.
+
+##### Configuration to support Data Security
+It is developer's responsibility to design and configure their LoopBack application using the [GDPR Privacy By Design](https://www.eugdpr.org/more-resources-1.html) as guidelines to support Data Security.
+
+___________________________________________________________________________________________________
+### Data Life Cycle
+
+LoopBack is a development framework that does not collect any data nor does it send any of it back to IBM.
+The LoopBack framework facilitates the connection to backend resources, such as databases and Web services, where identity data is stored.  However, the framework itself does not store actual data.
+
+_Important_:  LoopBack 4 is in developer preview.  It is still under change that how the application configuration data reside and being used in a LoopBack 4 application.  More information will be provided once the code has stablized.
```

Comment:
`LoopBack 4 is in developer preview. How its application configuration data reside and is being used may be subjected to change. More inform...`
I think grammatically this will flow better

Reply:
yes! subject to change is the phrase i was looking for!